programs/ssl/ssl_client2.c

```diff
@@ -134,9 +135,16 @@ int main( void )
 #define USAGE_IO \
     "    No file operations available (MBEDTLS_FS_IO not defined)\n"
 #endif /* MBEDTLS_FS_IO */
-#else
+#else /* MBEDTLS_X509_CRT_PARSE_C */
 #define USAGE_IO ""
 #endif /* MBEDTLS_X509_CRT_PARSE_C */
+#if defined(MBEDTLS_USE_PSA_CRYPTO) && defined(MBEDTLS_X509_CRT_PARSE_C)
+#define USAGE_OPAQUE \
+    "    key_opaque=%%d       Handle your private key as if it were opaque\n" \
```

Comment:
I'd rather say "As if it was". Was is used if the situation is probable, and "were" when it's pretty much impossible (simplyfying). 
If I were a rich man... ;)

Reply:
That's not a rule I've ever heard of and I suspect that you're confusing it with a slightly different rule, which is that it's *were* (subjunctive) is for hypotheticals and counter-factuals, whereas *was* is for something that plausibly happened in the past. By this rule, the correct word is *were*, since the private key is not in fact opaque.

However the subjunctive in English is somewhat old-fashioned. It's more common in the US, but losing ground. I'd use *was*, but just because it's more likely to be understood, not because of some difference in meaning.

Reply:
Well, currently it is impossible to provide a truly opaque key... well, I think it depends on your definition of possible, because of course we could add such an option later, but that's currently not the case. I think it's similar to "if I were rich": it's possible that this might become true in the future, but we're insisting that it's currently not the case at all. I think both could make sense for this option, but I felt more like insisting that the private key isn't really opaque so far.